tests/TestSuite/Mutant/ResultTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+*
+* @category   Humbug
+* @package    Humbug
+* @copyright  Copyright (c) 2015 Pádraic Brady (http://blog.astrumfutura.com)
+* @license    https://github.com/padraic/humbug/blob/master/LICENSE New BSD License
+* @author     Thibaud Fabre
+*/
+namespace Humbug\Test\TestSuite\Mutant;
+
+use Humbug\TestSuite\Mutant\Result;
+
+class ResultTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetStatusCodeReturnsCorrectCode()
+    {
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, true, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(false, false, true));
+        $this->assertEquals(Result::TIMEOUT, Result::getStatusCode(true, false, true));
```

Comment:
I think that problem is not here but in `Collector` class in fact. It is just propagated here somehow.
If `Collector` had methods like `collectTimeouts`, `collectKilled`, collectEscaped and `collectError` this class would disapear.

But if you don`t want to destroy your desk with your head I would leave it that way. But leave some kind of TODO in comment.